hadoop-build-tools/src/main/resources/checkstyle/checkstyle.xml

```diff
@@ -121,7 +121,7 @@
         <!-- See http://checkstyle.sf.net/config_import.html -->
         <module name="IllegalImport">
           <property name="regexp" value="true"/>
-          <property name="illegalPkgs" value="^sun\.[^.]+, ^com\.google\.common\.[^.]+"/>
+          <property name="illegalPkgs" value="sun, ^com\.google\.common"/>
```

Comment:
The `^` before com.google.common can be removed as well?
```suggestion
          <property name="illegalPkgs" value="sun, com\.google\.common"/>
```

Reply:
Thanks @aajisaka . you are right. It turned out that the editor has not saved my latest changes before committing.
It should be good to go now.